plugin-maven/src/main/java/com/diffplug/spotless/maven/AbstractSpotlessMojo.java

```diff
@@ -120,6 +121,9 @@
 	@Parameter(property = "spotlessFiles")
 	private String filePatterns;
 
+	@Parameter(property = LicenseHeaderStep.spotlessSetLicenseHeaderYearsFromGitHistory)
+	private String setLicenseHeaderYearsFromGitHistory;
```

Comment:
Can this field/parameter be boolean instead of string?
I think this could simplify type signatures (`boolean` instead of `Optional<String>`) and the check in `LicenseHeader`

Reply:
I think there's a decent chance that we'll allow values besides "true" in the future.  For example, you might want to exclude certain commits, say formatting commits, from the git history that you care about, and this is the pipe that we'll put that through.